app/controllers/api/admin/articles_controller.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class Api::Admin::ArticlesController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    if current_user.role == 'editor'
```

Comment:
Does this mean we don't have to include role in sign-in response?

Reply:
No, I guess we have to include it for conditional rendering of UI...

Reply:
The :role is automatically included in the sign_in response

Reply:
Nice. I was actually wondering if it would be.

Reply:
It automatically adds it